src/py42/clients/detectionlists/high_risk_employee.py

```diff
@@ -112,12 +109,12 @@ def _get_high_risk_employees_page(
         return self._session.post(uri, data=json.dumps(data))
 
     def get_all(self, filter_type=u"OPEN", sort_key=None, sort_direction=None):
-        """Search High Risk employee list. Filter results by filter_type.
+        """Search High Risk Employee list. Filter results by filter_type.
```

Comment:
`High Risk Employees` to be consistent, right?

Reply:
@annie-payseur didn't call out the need for pluralization on HRE anywhere.

Reply:
But if you look at other IX docs, https://duckduckgo.com/?q=high+risk+employees+code42&ia=web,
you will see it is plural.

Reply:
This is tricky - I pluralized HRE in the guide because that matches the console UI and therefore our IX docs, however, I realize it's singular in the detection list management APIs. Now I'm not sure which direction to go here.

Reply:
As long as they are consistent, either works! Probably easier to make them singular since most places in the code and docs, they are already singular.